nemo/collections/asr/modules/conv_asr.py

```diff
@@ -574,12 +575,16 @@ def __init__(
         self.decoder_layers.append(nn.Conv1d(self.feat_hidden, self.feat_out, kernel_size=1, bias=True))
 
         self.decoder_layers = nn.Sequential(*self.decoder_layers)
+        self.apply_softmax = apply_softmax
 
         self.apply(lambda x: init_weights(x, mode=init_mode))
 
     @typecheck()
     def forward(self, encoder_output):
-        return self.decoder_layers(encoder_output).transpose(-2, -1)
+        out = self.decoder_layers(encoder_output).transpose(-2, -1)
+        if self.apply_softmax:
+            out = torch.nn.functional.log_softmax(out, dim=-1)
```

Comment:
You need to update the output_types neural type of this module - when the flag is set to false, it returns logits, when true, log probs. It is important to update the semantics of this module and any downstream modules that accept the output of this class.